Quotient/connection.h

```diff
@@ -947,6 +950,9 @@ public Q_SLOTS:
     Room* provideRoom(const QString& roomId,
                       Omittable<JoinState> joinState = none);
 
+    //! Add room to the connection
```

Comment:
Please also mention that this is only for tests and not useful in normal operation